### PR TITLE
Set timeout for sending messages to redis.

### PR DESCRIPTION
**Pull request by rantan**

### CODE REVIEW

File: src/net.rs [hunk 1]
```diff
@@ -270,6 +270,7 @@ impl RedisManager {
                     let conn = client.get_connection()?;
                     thread::sleep(Duration::from_millis(500));
 
+                    conn.set_write_timeout(Some(Duration::from_secs(5)))?;
```
Comment (azuchi) [inline src/net.rs]: Do you need to set `set_read_timeout` too?

Comment (rantan) [reply src/net.rs]: No, this connection is only for sending message.

Comment (azuchi) [reply src/net.rs]: yes. What I mean is that it is need to set set_read_timeout elsewhere to receive messages from Redis.

Comment (rantan) [reply src/net.rs]: It is because receiving message connection doesn't have timeout. It's always waiting messages.

**Pull request merged**